File: FindPlacesNearMe/FindPlacesNearMe/Models/UserLocation+CoreDataProperties.m
//
//  UserLocation+CoreDataProperties.m
//  FindPlacesNearMe
//
//  Created by Mobile OS on 4/3/2020.
//  Copyright © 2020 myproject. All rights reserved.
//
//

#import "UserLocation+CoreDataProperties.h"

@implementation UserLocation (CoreDataProperties)

+ (NSFetchRequest<UserLocation *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"UserLocation"];
}

@dynamic latitude;
@dynamic longitude;
@dynamic searches;

@end

File: FindPlacesNearMeSwift/FindPlacesNearMeSwift/Model/UserLocation+CoreDataProperties.swift
//
//  UserLocation+CoreDataProperties.swift
//  FindPlacesNearMeSwift
//
//  Copyright © 2020 Amor Rio. All rights reserved.
//

import Foundation
import CoreData


extension UserLocation {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<UserLocation> {
        return NSFetchRequest<UserLocation>(entityName: "UserLocation")
    }

    @NSManaged public var latitude: Double
    @NSManaged public var longitude: Double
    @NSManaged public var searches: NSSet?

}

// MARK: Generated accessors for searches
extension UserLocation {

    @objc(addSearchesObject:)
    @NSManaged public func addToSearches(_ value: UserLocationSearch)

    @objc(removeSearchesObject:)
    @NSManaged public func removeFromSearches(_ value: UserLocationSearch)

    @objc(addSearches:)
    @NSManaged public func addToSearches(_ values: NSSet)

    @objc(removeSearches:)
    @NSManaged public func removeFromSearches(_ values: NSSet)

}
